doc/quick_tutorial.rst

```diff
@@ -63,7 +73,8 @@ buttons.
 
 Step 5 – Export your Video
 ---------------------------
-Once you are happy with your photo slide-show video, the next step is to export your video.
+
+Once you are happy with your video, the next step is to export your video.
```

Comment:
Ditto here, re: redundancy. Perhaps "Once you are happy with your layout...", or "Once you are happy with your project..."?